x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_group_main_statistics.ts

```diff
@@ -157,15 +159,17 @@ export async function getErrorGroupMainStatistics({
 
   const errorGroups =
     response.aggregations?.error_groups.buckets.map((bucket) => {
+      const normalizedFields = errorGroupMainStatisticsMapping(bucket.sample.hits.hits[0].fields);
+
       return {
         groupId: bucket.key as string,
-        name: getErrorName(bucket.sample.hits.hits[0]._source),
-        lastSeen: new Date(bucket.sample.hits.hits[0]._source['@timestamp']).getTime(),
+        name: getErrorName({ error: normalizedFields?.error } as APMError),
+        lastSeen: new Date(normalizedFields?.['@timestamp'] as string).getTime(),
         occurrences: bucket.doc_count,
-        culprit: bucket.sample.hits.hits[0]._source.error.culprit,
-        handled: bucket.sample.hits.hits[0]._source.error.exception?.[0].handled,
-        type: bucket.sample.hits.hits[0]._source.error.exception?.[0].type,
-        traceId: bucket.sample.hits.hits[0]._source.trace?.id,
+        culprit: normalizedFields?.error?.culprit,
+        handled: normalizedFields?.error?.exception[0]?.handled,
+        type: normalizedFields?.error?.exception[0]?.type,
+        traceId: normalizedFields?.trace?.id,
```

Comment:
nit: Optional chaining is only needed for `normalizedFields`. If the object is not undefined, then the rest is defined
```suggestion
        culprit: normalizedFields?.error.culprit,
        handled: normalizedFields?.error.exception[0].handled,
        type: normalizedFields?.error.exception[0].type,
        traceId: normalizedFields?.trace.id,
```